go/oasis-test-runner/scenario/e2e/runtime.go

```diff
@@ -102,6 +107,10 @@ func (sc *runtimeImpl) Parameters() *flag.FlagSet {
 	f.StringVar(&sc.runtimeBinaryDir, cfgRuntimeBinaryDir, sc.runtimeBinaryDir, "path to the runtime binaries directory")
 	f.StringVar(&sc.runtimeLoader, cfgRuntimeLoader, sc.runtimeLoader, "path to the runtime loader")
 	f.StringVar(&sc.TEEHardware, cfgTEEHardware, sc.TEEHardware, "TEE hardware to use")
+	// XXX: ideally iasMock would default to true, but the Clone/Parameters
```

Comment:
Hm, can you explain a bit more what would need to change? cc @matevz

Reply:
`Parameters()` always returns a new flagset (which feels wrong, i think new flagset should only be created in `Clone()` - but didn't want to change that in this PR).

The above in conjunction with:
https://github.com/oasislabs/oasis-core/blob/a408af740b9ba84b14743a821572a702dc59f557/go/oasis-test-runner/cmd/root.go#L133-L135

The already overridden parameter gets replaced with the default value every time this is called, if the existing var isn't passed in as `default`. This can be fixed by calling `sCloned.Parameters()` once and storing the returned flagset. 

But didn't check if any other code expects the same - and a more proper fix would be `Parameters()` not creating a new flagset i think.

Reply:
Ok, feel free to convert your comment to an issue and reference it in the comment ;-)